packages/pds/src/services/index.ts

```diff
@@ -19,6 +19,8 @@ import { BackgroundQueue } from '../event-stream/background-queue'
 import { Crawlers } from '../crawlers'
 import { LabelCache } from '../label-cache'
 import { ContentReporter } from '../content-reporter'
+import { LocalService } from './local'
+import { AtpAgent } from '@atproto/api'
```

Comment:
Same tiny baby nit: could be nice to hoist this up near the other `@atproto/*` deps.